roles/lib_utils/action_plugins/sanity_checks.py

```diff
@@ -2,11 +2,14 @@
 Ansible action plugin to ensure inventory variables are set
 appropriately and no conflicting options have been provided.
 """
+import json
 import re
+import urlparse
```

Comment:
In order to support python3 need to import this from ansible's six library. See https://github.com/openshift/openshift-ansible/blob/master/roles/lib_utils/filter_plugins/oo_filters.py#L29